libbeat/docs/release.asciidoc

```diff
@@ -30,3 +30,12 @@ This section summarizes the changes in each release.
 --
 
 include::../../CHANGELOG.asciidoc[]
+
+== Combined release notes
```

Comment:
So to clarify my earlier comment, I think we need to have only the 5.0 release notes in this document. So the nav would look like:

Release Notes
   Beats version 5.0.0

Then when 5.0.1 comes out, we'll add another section. And if people want to see a more granular description of the changes, they can look at the changelog.

Maybe we keep to the automated format when we create pre-release software, but then we curate the list and create combined release notes at GA. Does that make sense?